hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -8,14 +8,285 @@ title: "OTP Controller Technical Specification"
+  - Some partitions can be locked statically (by computing and storing an associated digest in OTP), others can be read locked at runtime via a CSRs.
+- Write lockability
+  - This controls whether a partition is locked and prevented from future updates.
+  - A locked partition is stored alongside a 32b digest (truncated hash) to be used later for integrity verification.
+- Integrity Verification
+  - Once a partition is locked, it can undergo periodic verification (time-scale configurable by software)
+This verification takes two forms, partition integrity checks, and storage consistency checks.
+
+Since the OTP is memory-like in nature (it only outputs a certain number of bits per address location), some of the logical partitions are buffered in registers for instantaneous and parallel access by hardware.
+This is a critical point, as after power-up, OTP contents are stored in flip flops and sourced to the system.
+They are **NOT** directly sourced from the OTP macro itself.
+Thus the security of both volatile (OTP controller) and non-volatile (OTP IP) storage becomes important.
+
+### Collateral
+
+The OTP controller for OpenTitan contains the five logical partitions shown below.
+
+Partititon     | Collateral                 | Secret | Buffered | WR Lockable  | RD Lockable  | Usage
+---------------|----------------------------|--------|----------|--------------|--------------|----------------------------------------------------------------
+CREATOR_SW_CFG | Device manufacturing data  | no     | no       | yes (digest) |   yes (CSR)  | Device-specific calibration (Clock, LDO, RNG, device identity).
+OWNER_SW_CFG   | Software configuration     | no     | no       | yes (digest) |   yes (CSR)  | Data that changes software behavior, specifically in the ROM. <br> E.g., enabling defensive features in ROM or selecting failure modes if verification fails.
+HW_CFG         | Hardware configuration     | no     | yes      | yes (digest) |      no      | Hardware configuration bits used to hardwire specific hardware functionality. <br> E.g., raw entropy accessibility or flash scrambling bypass range.
+LIFE_CYCLE     | Life-cycle state, counters | no     | yes      |     no       |      no      | Life-cycle related bits (ADD link). **Note**, this partition cannot be locked as the life-cycle state needs to be able to advance to RMA in the field.
+SECRET         | Tokens and Keys            | yes    | yes      | yes (digest) | yes (digest) | Device-unique secrets.
+
+Generally speaking, the production life cycle of a device is split into 5 stages "Manufacturing" -> "Calibration and Testing" -> "Provisioning" -> "Mission" -> "RMA".
+OTP values are usually programmed during "Calibration and Testing", "Provisioning" and "RMA" stages, as explained below.
+A detailed listing of all the collateral and the corresponding memory map can be found in the Programmer's Guide Section further below.
```

Comment:
maybe it could be good to include a link to the `programmers guide` section

Reply:
Yes I've tried that but without success... for some reason I did not manage to create a link that HUGO would render correctly...

Reply:
ah now it works - forgot to strip away the apostrophe :D